code/src/ssd1306.cpp

```diff
@@ -56,4 +59,10 @@ namespace r2d2::display {
         // update the local cursor
         cursor.x++; // TODO: cursor fallthrough
     }
+
+
+    uint16_t ssd1306_i2c_c::color_to_pixel(const hwlib::color &c) {
```

Comment:
I assume this is a uint16_t because it is an inheritance, it should be explained in doxygen why it returns a bool.

Reply:
in the hpp ofcourse